test/testrecipes/test-terraform-recipes/azure-storage/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source = "hashicorp/azurerm"
-      version = "~> 3.0.0"
+      version = "~> 3.114.0"
```

Comment:
~> 3.0.0 will only match 3.0.*, but workload identity for azurerm needs greater than 3.7. 3.114 is the latest version as of today